graphql-java-support/src/test/java/com/apollographql/federation/graphqljava/TestUtils.java

```diff
@@ -6,13 +6,13 @@
 import java.util.stream.Collectors;
 
 final class TestUtils {
-    private TestUtils() {
-    }
+
+    private static final String BASE_LINE_SEPARATOR = "\n";
 
     static String readResource(String name) {
         InputStream is = SchemaUtils.class.getResourceAsStream(name);
         assert is != null;
         BufferedReader reader = new BufferedReader(new InputStreamReader(is));
-        return reader.lines().collect(Collectors.joining(System.lineSeparator()));
+        return reader.lines().collect(Collectors.joining(BASE_LINE_SEPARATOR));
```

Comment:
OK, so what this does is normalize any line endings on disk to unix newline. That seems reasonable.